src/server/multi_test.cc

```diff
@@ -50,6 +50,21 @@ class MultiTest : public BaseFamilyTest {
   MultiTest() : BaseFamilyTest() {
     num_threads_ = kPoolThreadCount;
   }
+
+  void SetUp() override {
```

Comment:
gtest should automatically restore all flag values upon test finish, see here:
https://github.com/google/googletest/blob/cc366710bbf40a9816d47c35802d06dbaccb8792/googletest/src/gtest.cc#L2460
Does this not work for some reason?

Reply:
Looking slightly more into this, it looks like gtest tries to use absl when it finds it, so perhaps it can't for some reason and then the flags it uses are incompatible. It's slightly complicated. LMK if I can be of help debugging this.

Reply:
And... if we can't get it to work (hopefully we can though), we should probably use `absl::FlagSaver` (see [here](https://chromium.googlesource.com/external/github.com/abseil/abseil-cpp/+/refs/heads/master/absl/flags/reflection.h)). Possibly at a higher level (like in `BaseFamilyTest`).

Reply:
Ok I will check why it does not restores the flags

Reply:
@chakaz I am using flag saver now, please take a look
@romange will try to use more updated gtest and passing the ABSL_HAS_GTEST later on